src/libinjection_sqli.h

```diff
@@ -20,20 +20,24 @@ extern "C" {
 #include <string.h>
 
 enum sqli_flags {
-    FLAG_NONE            = 0
-    , FLAG_QUOTE_NONE    = 1   /* 1 << 0 */
-    , FLAG_QUOTE_SINGLE  = 2   /* 1 << 1 */
-    , FLAG_QUOTE_DOUBLE  = 4   /* 1 << 2 */
-
-    , FLAG_SQL_ANSI      = 8   /* 1 << 3 */
-    , FLAG_SQL_MYSQL     = 16  /* 1 << 4 */
+    FLAG_NONE = 0,
+    FLAG_QUOTE_NONE = 1 /* 1 << 0 */
+    ,
```

Comment:
Should the comma be next to the value here?

Reply:
Well... yes :) But the format was not changed by `clang-format`, so I guess it is allowed in the standard... changed it anyway.